cmake/CMakeLists.txt

```diff
@@ -798,7 +798,7 @@ foreach(target_name onnxruntime_common onnxruntime_graph onnxruntime_framework o
 endforeach()
 
 foreach(provider_name ${ONNXRUNTIME_PROVIDER_NAMES})
-  if(NOT provider_name STREQUAL "cpu")
+  if(NOT provider_name STREQUAL "cpu" AND NOT provider_name STREQUAL "winml")
```

Comment:
Which flag the winml doesn't support?

Reply:
onnxruntime_providers_winml doesnt have a target that gets built. Only headers.